Droidcon-Boston/app/src/main/java/com/mentalmachines/droidcon_boston/data/FirebaseDatabase.kt

```diff
@@ -78,9 +78,9 @@ open class FirebaseDatabase {
 
         data class Answer(
                 var answer: String = "",
-                var photoLink: String = "",
-                var mapLink: String = "",
-                var otherLink: String = ""
+                var photoLink: String? = "",
```

Comment:
Why do these variables need to allow `null` values? In code, check for empty string should do the work.

Reply:
These actually were nullable before (i.e., the '= ""' default value was *not* there)
No idea who changed it...Android Studio's Annotate feature is broken :-(

Reply:
switching it to be non-nullable..don't think it's ever null.

Reply:
it actually broke the FAQ reskin screens when I merged in because I was expecting nulls...